nano/node/json_handler.cpp

```diff
@@ -937,10 +938,18 @@ void nano::json_handler::accounts_representatives ()
 				continue;
 			}
 		}
-		representatives.put (account_from_request.second.data (), boost::format ("error: %1%") % ec.message ());
+		debug_assert (ec);
+		errors.put (account_from_request.second.data (), ec.message ());
 		ec = {};
 	}
-	response_l.add_child ("representatives", representatives);
+	if (!representatives.empty ())
```

Comment:
Considering the ptree issue with serialising empty arrays as empty strings, I think this is the less-wrong way to do it. RPC users can test if the array member exists before iterating. In the future when the serialisation is fully conformant, the check could be removed. Otherwise they need to handle different types which is undesirable.